<ui version="4.0" >
 <class>ParametersForm</class>
 <widget class="QWidget" name="ParametersForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QVBoxLayout" name="verticalLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="noEffectButton" >
        <property name="text" >
         <string>No Effect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="grayscaleButton" >
        <property name="text" >
         <string>Grayscale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="tintButton" >
        <property name="text" >
         <string>Tint</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="optionsWidget" >
     <layout class="QGridLayout" name="gridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Red</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSlider" name="redSlider" >
        <property name="maximum" >
         <number>255</number>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QSpinBox" name="redSpinBox" >
        <property name="maximum" >
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Green</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSlider" name="greenSlider" >
        <property name="minimum" >
         <number>0</number>
        </property>
        <property name="maximum" >
         <number>255</number>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
        <property name="sliderPosition" >
         <number>0</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QSpinBox" name="greenSpinBox" >
        <property name="maximum" >
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Blue</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSlider" name="blueSlider" >
        <property name="minimum" >
         <number>0</number>
        </property>
        <property name="maximum" >
         <number>255</number>
        </property>
        <property name="value" >
         <number>99</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QSpinBox" name="blueSpinBox" >
        <property name="maximum" >
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="widget_2" >
     <layout class="QFormLayout" name="formLayout" >
      <property name="margin" >
       <number>1</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Composition Mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="compositionModeBox" >
        <item>
         <property name="text" >
          <string>Screen</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Overlay</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Multiply</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>redSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>214</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel" >
     <x>271</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>214</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel" >
     <x>271</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>214</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel" >
     <x>271</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>271</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel" >
     <x>214</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>271</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel" >
     <x>214</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>271</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel" >
     <x>214</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tintButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>redSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>40</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel" >
     <x>62</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tintButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>greenSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>58</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel" >
     <x>63</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tintButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>blueSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>80</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel" >
     <x>93</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tintButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>redSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>239</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel" >
     <x>246</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tintButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>greenSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>262</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel" >
     <x>263</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tintButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>blueSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>230</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel" >
     <x>251</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
